contributing-guides/translation-templates/common-arguments.md

```diff
@@ -7,7 +7,7 @@ Only the left-alignment of the header gets lost and has to be re-added again (`|
 
 | en    | path/to/file         | path/to/directory      | path/to/file_or_directory         | package   | username          |
 |:------|:---------------------|:-----------------------|:----------------------------------|:----------|:------------------|
-| ar    |                      |                        |                                   |           |                   |
+| ar    | المسار/إلى/الملف     | المسار/إلى/الدليل        | المسار/إلى/الملف_أو_الدليل           | حزمة          | اسم_المستخدم      |
```

Comment:
Could you try to align the `|` with the rows above and below.

Reply:
Sure, thought that I must use that website only.

Reply:
it is hard to do that in here. Arabic gets messy because of RTL etc.
The first column is actually the "username" field in the Arabic translation.

Reply:
> it is hard to do that here. Arabic gets messy because of RTL etc. The first column is actually the "username" field in the Arabic translation.

Alignment issues aside, I don't think we should be rearranging the table. But rearranging it isn't possible the way Arabic is written (i.e from right to left). I think the best solution would be adding the line in a new (reversed) table below this one @navarroaxel What do you think?